pay-api/tests/unit/api/test_payment_request.py

```diff
@@ -316,9 +317,13 @@ def test_payment_creation_with_routing_slip(session, client, jwt, app):
     assert schema_utils.validate(rv.json, 'invoice')[0]
 
 
-def test_payment_creation_with_existing_routing_slip(client, jwt):
+@pytest.mark.parametrize('payment_requests', [
+    get_payment_request(),
+    get_payment_request_without_bn()
+])
+def test_payment_creation_with_existing_routing_slip(client, jwt, payment_requests):
     """Assert that the endpoint returns 201."""
-    claims = get_claims(roles=[Role.FAS_CREATE.value, Role.FAS_SEARCH.value, Role.STAFF.value])
+    claims = get_claims(roles=[Role.FAS_CREATE.value, Role.FAS_SEARCH.value, Role.STAFF.value, 'make_payment'])
```

Comment:
Had to give make_payment to this test user since in this case auth is not invoked to get the staff roles.Change only in test.